packages/riipen-ui/src/components/Avatar.jsx

```diff
@@ -25,6 +25,11 @@ class Avatar extends React.Component {
      */
     classes: PropTypes.array,
 
+    /**
+     * The color of the background for the Avatar
+     */
+    backgroundColor: PropTypes.oneOf(["grey", "white"]),
```

Comment:
Sorry I should have caught this earlier. When we use greys in our props, we have the standard of using the fully qualified grey like "grey200", "grey600", or "greyA400", so I think in this case, we should probably use `grey200`.